streaming/src/test/scala/org/apache/spark/streaming/StreamingContextSuite.scala

```diff
@@ -575,8 +575,6 @@ class StreamingContextSuite extends SparkFunSuite with BeforeAndAfter with TimeL
 
   test("getActive and getActiveOrCreate") {
     require(StreamingContext.getActive().isEmpty, "context exists from before")
-    sc = new SparkContext(conf)
```

Comment:
why this change?

Reply:
Because the test stops the underlying streaming context and thus the spark context, and you can't add listeners to the bus after that. Thus, it was failing.